backend/src/test/java/com/funeat/acceptance/product/ProductAcceptanceTest.java

```diff
@@ -41,486 +59,512 @@
 @SuppressWarnings("NonAsciiCharacters")
 class ProductAcceptanceTest extends AcceptanceTest {
 
-    public static final int PAGE_SIZE = 10;
+    private static final Long PAGE_SIZE = 10L;
+    private static final Long FIRST_PAGE = 0L;
 
     @Nested
-    class 가격_기준_내림차순으로_카테고리별_상품_목록_조회 {
-
-        @Test
-        void 상품_가격이_서로_다르면_가격_기준_내림차순으로_정렬할_수_있다() {
-            // given
-            final Long categoryId = 카테고리_추가_요청(간편식사);
-            final Product product1 = new Product("삼각김밥1", 1000L, "image.png", "맛있는 삼각김밥1", 간편식사);
-            final Product product2 = new Product("삼각김밥2", 2000L, "image.png", "맛있는 삼각김밥2", 간편식사);
-            final Product product3 = new Product("삼각김밥3", 1500L, "image.png", "맛있는 삼각김밥3", 간편식사);
-            final Product product4 = new Product("삼각김밥4", 1200L, "image.png", "맛있는 삼각김밥4", 간편식사);
-            final Product product5 = new Product("삼각김밥5", 2300L, "image.png", "맛있는 삼각김밥5", 간편식사);
-            final Product product6 = new Product("삼각김밥6", 1700L, "image.png", "맛있는 삼각김밥6", 간편식사);
-            final Product product7 = new Product("삼각김밥7", 1800L, "image.png", "맛있는 삼각김밥7", 간편식사);
-            final Product product8 = new Product("삼각김밥8", 800L, "image.png", "맛있는 삼각김밥8", 간편식사);
-            final Product product9 = new Product("삼각김밥9", 3100L, "image.png", "맛있는 삼각김밥9", 간편식사);
-            final Product product10 = new Product("삼각김밥10", 2700L, "image.png", "맛있는 삼각김밥10", 간편식사);
-            final Product product11 = new Product("삼각김밥11", 300L, "image.png", "맛있는 삼각김밥11", 간편식사);
-            final List<Product> products = List.of(product1, product2, product3, product4, product5, product6, product7,
-                    product8, product9, product10, product11);
-            복수_상품_추가_요청(products);
-
-            // when
-            final var response = 카테고리별_상품_목록_조회_요청(categoryId, "price", "asc", 0);
-
-            // then
-            STATUS_CODE를_검증한다(response, 정상_처리);
-            페이지를_검증한다(response, (long) products.size(), 0L);
-            카테고리별_상품_목록_조회_결과를_검증한다(response,
-                    List.of(product11, product8, product1, product4, product3, product6, product7, product2, product5,
-                            product10));
+    class getAllProductsInCategory_테스트 {
+
+        @Nested
+        class 성공_테스트 {
+
+            @Nested
+            class 가격_기준_내림차순으로_카테고리별_상품_목록_조회 {
```

Comment:
```suggestion
    @Nested
    class getAllProductsInCategory_성공_테스트 {

        @Nested
        class 가격_기준_내림차순으로_카테고리별_상품_목록_조회 {
```

이렇게 맨위 getAllProduct~~~ + 성공(실패) 테스트를 합치는 건 어떻게 생각하시나요?
ide 로 코드를 확인하는데 생각보다 뎁스가 깊어서 코드가 오른쪽으로 많이 밀려서요

Reply:
좋아요~~ 그거는 생각 못했는데 저렇게 수정하면 ide에서 코드 보기 더 수월해지겠네여